src/stdlib/src/lib.rs

```diff
@@ -1,15 +1,16 @@
-//! stdlib implementation for Redox, following http://pubs.opengroup.org/onlinepubs/7908799/xsh/stdlib.h.html
```

Comment:
Why was this comment line moved?

Reply:
The compiler was complaining about it. It said no doc comments before the #![]s

Reply:
Why does the line have three slashes instead of two now?

Reply:
I guess I did that by accident

Reply:
If you change it back to two can it move back to the top of the file?

Reply:
Yes.

Reply:
:tada: Sweet! :+1: